src/Rector/Contrib/PHPUnit/SpecificMethod/AssertPropertyExistsRector.php

```diff
@@ -73,7 +82,7 @@ public function isCandidate(Node $node): bool
             return false;
         }
 
-        if (! $this->methodCallAnalyzer->isMethods($node, array_keys($this->renameMethodsMap))) {
+        if (! $this->methodCallAnalyzer->isMethods($node, ['assertTrue', 'assertFalse'])) {
```

Comment:
We can use either `$renameMethodsWithClassMap` or `$renameMethodsWithObjectMap` here

Reply:
Since there are now two maps, it puts me into position to choose one of those. I find this clearer and more efficient as I don't have to transform keys to array.